### PR TITLE
User and Orgs Models

### LINKED ISSUE #95: Add models for users and organizations

### PR DESCRIPTION
**Pull request by FoseFx**
Closes #95

### CODE REVIEW

File: db/migrations/000002_auth.up.sql [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+);
+
+ALTER TABLE emergency_rooms
+ADD COLUMN IF NOT EXISTS
+owned_by UUID NOT NULL;
+
+
+ALTER TABLE emergency_rooms
+ADD CONSTRAINT emergency_rooms_owned_by_foreign_key_constraint
+FOREIGN KEY (owned_by) REFERENCES organizations(id);
```
Comment (MaxSchaefer) [inline db/migrations/000002_auth.up.sql:36]: Shouldn't `owned_by` actually be an attribute on the `EmergencyRoom` model as well?

**Review by MaxSchaefer — CHANGES_REQUESTED** [review]

**Comment by FoseFx** [comment]
This is the ERD now btw: https://github.com/helpwave/helpwave/pull/9

File: models/room.go [hunk 2]
```diff
@@ -15,5 +15,7 @@ type EmergencyRoomBase struct {
 type EmergencyRoom struct {
 	ID uuid.UUID
 	EmergencyRoomBase
-	Departments []Department `gorm:"many2many:rooms_have_departments" json:"-"`
+	OrganizationID uuid.UUID
+	Organization   Organization
```
Comment (MaxSchaefer) [inline models/room.go:19]: Shouldn't there be a GORM annotation here? Take a look at https://github.com/helpwave/rest-api/blob/main/models/emergency.go#L14.

Comment (MaxSchaefer) [reply models/room.go:19]: Nice, thanks for the clarification 👍

**Review by MaxSchaefer — APPROVED** [review]
Let's get this auth thing started 🎉

**Pull request merged**